src/main/archetype/ui.apps/src/main/content/jcr_root/apps/fd/af/theme-clientlibs/adaptiveform.theme.healthcare/.content.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    categories="[adaptiveform.theme.healthcare]"
```

Comment:
Please use customer appId as name in theme category

Reply:
${appId}.adaptiveform.theme.healthcare is new category name. Similar pattern for other themes.